docs/concepts/README.md

```diff
@@ -1,4 +1,4 @@
-# Presentation
+## Presentation
```

Comment:
According to markdown rules, headings should be increasing one by one.
It's semantic against display ;-)

You might have a mkdocs warning for that. Not sure if I have actived.
But not strong opinion.